src/lib/structures/Command.js

```diff
@@ -136,9 +136,11 @@ class Command {
 		/**
 		 * The extended help for the command
 		 * @since 0.0.1
-		 * @type {string}
+		 * @type {string|Function}
+		 * @param {CommandMessage} msg The command message mapped on top of the message used to trigger this command
```

Comment:
{KlasaMessage}

Reply:
in v0.5.0 it's not a command message mapped on top of message, it's just an extended message class instance. So: "The message used to trigger this command" would be a more apt description.